### PR TITLE
Testing & Changes

### PR DESCRIPTION
**Pull request by UtsavSomaiya**
- [x] User Id and Password are Wrong -> Incorrect credentials.
- [x] Remove link of Forgot password.
- [x] Display the product with less opacity if product stock not available.
- [x] Image upload testing.(move_uploaded_file).
- [x] Tax not pre-selected after form submit.(Testing & Chnages).
- [x] Simple edit icon in product list.
- [x] Same icons in category list.
- [x] Don't delete... -> Cannot delete this category as it is used by one ore more products.
- [x] Display existing discount.(Add the flat discount also).
- [x] Display that it's percentage or flat.
- [x] Validate percentage.
- [x] Sale details page product image should be displayed.(Same as above in product_list should be working so it can be working also).
- [x] Display Subtotal,Total_tax,Discount	Total on the sales details page.
- [x] Why underscore version on sale details page table columns.
- [x] Dashboard :- 

- Total sales count
- Total sales amount
- Total discount offered
- Total tax collected.

- [x] POS :

- Clear All button should work.
- Tax should be calculated after discount applied.

**Formatting is not proper so sorry for that. :smiling_face_with_tear:**

### CODE REVIEW

**Comment by UtsavSomaiya** [comment]
Done Now you can check this PR and Let me know.

File: admin/category/show_category.php [hunk 1]
```diff
@@ -32,8 +32,9 @@
 								<tr>
 									<td><?= $r['id']?></td>
 									<td><?= $r['name'] ?></td>
-									<td><a href="../category/edit_category.php?id=<?= $r['id']?>" class="btn btn-dark btn-icon-text">Edit<i class="ti-file btn-icon-append"></i></a></td>
-									<td><a href="javascript:alert_c(<?= $r['id']?>)" class="btn btn-outline-danger btn-fw">Delete</a>
+									<td><a href="../category/edit_category.php?id=<?= $r['id']?>"><img
+                                                src="/admin/images/1.png" /></a></a></td>
```
Comment (gauravmak) [inline admin/category/show_category.php:36]: Keep in a separate directory

File: index.php [hunk 2]
```diff
@@ -219,7 +238,6 @@ function toast() {
             <?php
               if (isset($_SESSION["msg"])) {
                   echo "toast()";
-                  //unset($_SESSION["msg"]);
```
Comment (gauravmak) [inline index.php:222]: ?

File: index.php [hunk 3]
```diff
@@ -1,15 +1,18 @@
 <?php
 require 'admin/layout/db_connect.php';
 session_start();
+$FLAT_DISCOUNT = 2;
+$PERCENTAGE_DISCOUNT = 1;
```
Comment (gauravmak) [inline index.php:5]: Use [constants](https://www.php.net/manual/en/language.constants.php)

**Review by gauravmak — CHANGES_REQUESTED** [review]

File: index.php [hunk 4]
```diff
@@ -1,8 +1,9 @@
 <?php
 require 'admin/layout/db_connect.php';
 session_start();
-$FLAT_DISCOUNT = 2;
-$PERCENTAGE_DISCOUNT = 1;
+define("discount", [
```
Comment (gauravmak) [inline index.php:4]: Should be in capital `discount`

**Review by gauravmak — CHANGES_REQUESTED** [review]

File: admin/product/show_product.php [hunk 5]
```diff
@@ -54,9 +53,8 @@
                                     <td><?= $product['stock'] ?></td>
                                     <td><img src="<?= '/admin/images/' . $product['image'] ?>"></td>
                                     <td><a href="../product/edit_product.php?id=<?= $product['id'] ?>"><img
-                                                src="/admin/images/icons8-edit.gif" /></a></td>
-                                    <td><a href="javascript:alert(<?= $product['id'] ?>)"><img
-                                                src="https://img.icons8.com/ios-glyphs/30/000000/filled-trash.png" /></a>
+                                                src="/admin/image/1.png" /></a></td>
```
Comment (gauravmak) [inline admin/product/show_product.php:56]: What is `1.png`? Keep proper image name

**Review by gauravmak — CHANGES_REQUESTED** [review]

**Pull request merged**